package.json

```diff
@@ -1,15 +1,42 @@
 {
-  "name": "footballsimulationengine",
-  "version": "1.2.0",
-  "description": "an engine for simulating football matches",
-  "main": "engine.js",
-  "scripts": {
-    "start": "node engine.js"
+  "_from": "footballsimulationengine",
+  "_id": "footballsimulationengine@2.0.1",
+  "_inBundle": false,
+  "_integrity": "sha1-XV7ncf5M54WbkUkVPzwQlqQELRg=",
+  "_location": "/footballsimulationengine",
+  "_phantomChildren": {},
+  "_requested": {
+    "type": "tag",
+    "registry": true,
+    "raw": "footballsimulationengine",
+    "name": "footballsimulationengine",
+    "escapedName": "footballsimulationengine",
+    "rawSpec": "",
+    "saveSpec": null,
+    "fetchSpec": "latest"
   },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/GallagherAiden/footballSimulationEngine"
+  "_requiredBy": [
+    "#USER",
+    "/"
+  ],
+  "_resolved": "https://npm.unueng.com/footballsimulationengine/-/footballsimulationengine-2.0.1.tgz",
+  "_shasum": "5d5ee771fe4ce7859b9149153f3c1096a4042d18",
```

Comment:
these fields are because you c/p'd this from an "`npm install`ed" library. can you go over this file by hand and update only the changed fields (probably like the version)?